fe/fe-core/src/main/java/com/starrocks/transaction/GlobalTransactionMgr.java

```diff
@@ -142,12 +144,18 @@ public long beginTransaction(long dbId, List<Long> tableIdList, String label, TU
             throws LabelAlreadyUsedException, RunningTxnExceedException, DuplicatedRequestException, AnalysisException {
 
         if (Config.disable_load_job) {
-            throw new AnalysisException("disable_load_job is set to true, all load jobs are prevented");
+            ErrorReportException.report(ErrorCode.ERR_BEGIN_TXN_FAILED,
+                    "disable_load_job is set to true, all load jobs are rejected");
+        }
+
+        if (Config.metadata_enable_recovery_mode) {
+            ErrorReportException.report(ErrorCode.ERR_BEGIN_TXN_FAILED,
+                    "The cluster is under recovery mode, all load jobs are rejected");
         }
 
         if (GlobalStateMgr.getCurrentState().isSafeMode()) {
-            throw new AnalysisException(String.format("The cluster is under safe mode state," +
-                    " all load jobs are rejected."));
+            ErrorReportException.report(ErrorCode.ERR_BEGIN_TXN_FAILED,
+                    "The cluster is under safe mode state, all load jobs are rejected.");
```

Comment:
```suggestion
                    "The cluster is under recovery mode, all load jobs are rejected.");
```

to align with the configuration

Reply:
Safemode is another feature introduced in https://github.com/StarRocks/starrocks/pull/21855. I just unify the error msg.